.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -22,8 +22,8 @@ is the best place for getting support.
 ### Description
 <!-- 
 Description of the issue
-Why is this change being proposed?
-Why should this be added to the codebase?
+Are there currently any workarounds?
```

Comment:
I'm not !00% sure these belong in the template, Maybe they'd be better off in a wiki or something. It seems weird, to me at least, to describe how to write the title of the issue inside the issue.

Reply:
If I trusted that everyone would be sure to read, and periodically refresh their memory of, the wiki then I'd agree.

I posted the wiki pages on Monday. You've filed a few issues an PRs since then. I'm curious - and I do not know the answer - did you read the wiki?

(Sorry for the little bit of a confrontational tone above. I'm shooting for wry because I _think_ I know what your answer will be but it's not landing quite right.)

Reply:
No worries and I will be honest, I did not read the wiki 😅 

It might be better to add a link in the block comments at the top as a 3rd section. something like

If you haven't opened an issue before, some good rules to follow are located here, but then again that pushes it somewhere else and isn't as, "in your face" for lack of a better term. 

Another alternative is to have a sample title that does all the things this post says. It won't be nearly as explicit but it should help to get the idea across. 
something like the example you posted above

Reply:
Perhaps a user story should be its own template?

I honestly don't care about writing these in a user story format, but rather making sure the template is structured in a way to get the necessary information about a bug/feature/etc. A user story defines a user, a goal, and a problem. We can easily capture this information with straight forward questions in the templates rather than enforcing a writing style.

Reply:
I think I want to put this into the description comment for improvement,
`<!-- who would want this change? -->`
Right now I'm trying to keep the amount of templates to a minimum so we don't have a different template for each type of issue, (especially when all the fields are the same). Does that seem like a good idea @peterhuffer

Reply:
I think we should stick with a small number of templates for a few days and see how it works. We can always add more later. I like your comment, Kyle.